### ISSUE TITLE
Hide Hit Column possible?

### ISSUE THREAD

**Issue by ExsiliumTrinity** [topeterk/HitCounterManager#10]
Would it be possible to hide either Hit Column or PB Column? I am currently trying to use this program to record Bosses Killed (Using the HitCounterYesNo.html). Issue is when I go down the list of bosses it auto checks on the Hits side as I go down and when i go back up it auto unchecks itself. Would it be possible to either: Manually select the check marks without auto selecting/deselecting? or Hide the Hits Column so only PB column shows? The PB Column works how i want it to, but i would like to be able to hide the Hits Column. If its something i would be able to edit on my end may i ask how? 

Thanks for the help!

**Comment by topeterk** [comment]
Hi, just to confirm I understand it correctly. So you are currently using the PB column because the Hits will mark the splits as done when one passed it. Means that, for example, you are doing the game in a random boss order and you just want to mark the ones you have done already?

Sounds like an interesting extension to the hit counter.

I have attached a design based on the YesNo file: [HitCounterBosses.zip](https://github.com/topeterk/HitCounterManager/files/2586385/HitCounterBosses.zip)
That should cover your case for now.

All the data from the application is available in the designs as a javascript object, so you can define any logic to the designs based on this data without changing the application.
For example in the attached design, you can switch between Hits column and PB column by just using the desired index in line 41...
Hits: `ShowCrossOrCheckMark(data.list[i][1] > 0 ? 0 : 1)`
PB: `ShowCrossOrCheckMark(data.list[i][2] > 0 ? 0 : 1)`
I also inverted the logic behind the hits in this line, so 0 hits means "boss is open" and 1 means "boss is done". I guess this makes it easier to use as by resetting the counter every boss is immediately marked as "open".

I don't know if you want that but when you don't want to see the selected split, you can also replace the condition `data.split_active == i` in line 39 with `false` and its gone.

Please try it and let me know if this was your intention (and if it worked of course) :wink:

**Comment by ExsiliumTrinity** [comment]
Nice! Thank You! You have no idea how long i have been looking for this. I have spent the last 4 days looking for this. I was on the verge of learning to code and make this myself. Everything i searched for kept coming back to Happy Hobs no hit run. (Maybe use his name as a search tag for the program, Just Kidding xD lol ) I only found this program thanks to randomly stumbling across TigerG92 on twitch and asked him.   

How hard would it to add another hotkey to go back to previous split. You have an hot key for next but not to go back. Basically allowing you to navigate back and forth:
Example:
-- Previous Split: Num 1
-- Next Split: Num 2
-- Got Hit: Num 3

I dont mind manually setting it with a mouse, but you know how Dark Souls games like to minimize when you leave the screen. 

Thanks again for your help and quick response!
